test/base/unit/MessagesSpec.scala

```diff
@@ -28,7 +28,7 @@ import play.api.mvc.Request
 abstract class MessagesSpec extends AnyWordSpec with GuiceOneAppPerSuite with Injector {
 
   override def fakeApplication(): Application = {
-    SharedMetricRegistries.clear()
+    SharedMetricRegistries.clear() //todo turn jvm metrics off for local.
```

Comment:
I presume you are suggesting that we ought also to be disabling JVM metrics when running tests? I think if we believe it's worth doing (I presume you are hoping for a performance gain?) then we should create a new Jira ticket.

Reply:
Just while running locally, this can be acheived via a varaoble in application conf, we can re-turn them on in app-config-base, however do we really need jvm metrics for local env. 
Would have the added benefit of not having to do this in tests